src/main/java/frc/trigon/robot/subsystems/turret/toohardturret/ToohardTurretIO.java

```diff
@@ -0,0 +1,25 @@
+package frc.trigon.robot.subsystems.turret.toohardturret;
+
+import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.controls.PositionVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.trigon.robot.subsystems.turret.TurretIO;
+import frc.trigon.robot.subsystems.turret.TurretInputsAutoLogged;
+
+public class ToohardTurretIO extends TurretIO {
+    private final TalonFX motor = ToohardTurretConstants.MOTOR;
+    private final MotionMagicVoltage positionRequest = new MotionMagicVoltage(0).withSlot(0);
```

Comment:
I LOVE this way of creating requests. So much so that this is the new conversion. 
Though your idea is very good, you didn't configure foc and there's no real need for `.withSlot()`